test/test_transforms.py

```diff
@@ -1406,18 +1411,14 @@ def test_trans_serial_env_check(self):
         env = TransformedEnv(SerialEnv(2, ContinuousActionVecMockEnv), StepCounter(10))
         check_env_specs(env)
 
-    def test_single_trans_env_check(self):
-        env = TransformedEnv(ContinuousActionVecMockEnv(), StepCounter(10))
-        check_env_specs(env)
-
     @pytest.mark.skipif(not _has_gym, reason="Gym not found")
     def test_transform_env(self):
         env = TransformedEnv(GymEnv(PENDULUM_VERSIONED), StepCounter(10))
         td = env.rollout(100, break_when_any_done=False)
         assert td["step_count"].max() == 9
         assert td.shape[-1] == 100
 
-    @pytest.mark.parametrize("step_key", ["step_count", ("other", "key")])
+    @pytest.mark.parametrize("step_key", ["step_count", "other-key"])
```

Comment:
why are we not testing the nested key anymore?

Reply:
@matteobettini we now restrict step_count to follow the done. Since it must match in shape etc there's not reason to allow it to be at a nested level. The idea is that we want to be able to match 
reset <=> done <=> truncated <=> step_count <=> is_init
So we need them all to be always at the same level.
Otherwise you could have a step_count at any level and it's hard to find where it comes from.

Reply:
got it thanks